pkg/backend/httpstate/token_source_test.go

```diff
@@ -26,11 +27,6 @@ import (
 )
 
 func TestTokenSource(t *testing.T) {
-	// TODO[pulumi/pulumi#16500] fix flaky test and unskip
-	t.Skip("Skipping flaky test: TODO[pulumi/pulumi#16500] to unskip")
-	if runtime.GOOS == "windows" {
-		t.Skip("Flaky on Windows CI workers due to the use of timer+Sleep")
-	}
```

Comment:
So this flakyness was due to network errors and not due to timing issues?

Reply:
Maybe... we're optimistically turning it back on to check.